src/platyPS/platyPS.psm1

```diff
@@ -599,6 +603,70 @@ function New-MarkdownAboutHelp
     }
 }
 
+function New-YamlHelp
+{
+    [CmdletBinding()]
+    [OutputType([System.IO.FileInfo[]])]
+    param(
+        [Parameter(Mandatory=$true,
+            Position=1,
+            ValueFromPipeline=$true,
+            ValueFromPipelineByPropertyName=$true)]
+        [string[]]$Path,
+
+        [Parameter(Mandatory=$true)]
+        [string]$OutputPath,
```

Comment:
I've updated this to use OutputFolder as the parameter name since that is the convention